packages/openid4vc/src/openid4vc-verifier/OpenId4VcSiopVerifierService.ts

```diff
@@ -553,9 +563,9 @@ export class OpenId4VcSiopVerifierService {
         // to fix that in Sphereon lib
         client_id: clientId,
         passBy: PassBy.VALUE,
```

Comment:
This one is also included in the final metadata. I think we can jsut remove it?

Reply:
that does not work. Many failing tests.

Reply:
Hmm okay, this should be fixed in sphereon's lib then.